src/components/canvas/autosave/index.tsx

```
@@ -0,0 +1,118 @@
+      </div>
+    );
+  }
+
+  switch (saveStatus) {
+    case 'saved':
+      return (
+        <div className="flex items-center">
+          <CheckCircle className="w-4 h-4" />
+        </div>
+      );
+    case 'saved':
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)